go.mod

```diff
@@ -72,6 +72,7 @@ require (
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
 	github.com/whs/nacl-sealed-box v0.0.0-20180930164530-92b9ba845d8d
 	github.com/yggdrasil-network/yggdrasil-go v0.4.0
+	github.com/yusufpapurcu/wmi v1.2.2 // indirect
```

Comment:
We definetly don't need that. Please make sure both go.mod and go.sum has no changes. Your code does not require any changes to the dependencies, specially not a windows support one.